packages/record-data/addon/-private/ts-interfaces/relationship-record-data.ts

```diff
@@ -30,6 +30,9 @@ export interface RelationshipRecordData extends RecordData {
   _implicitRelationships: {
     [key: string]: Relationship;
   };
+  __implicitRelationships: {
```

Comment:
Do you have a `Dict` type you could use here? `Dict<Relationship>`?

Reply:
We do